src/features/pastelDB/pastelDBLib.ts

```diff
@@ -1,5 +1,6 @@
 import { remote } from 'electron'
 import fs from 'fs'
+import { open } from 'fs/promises'
```

Comment:
I think the change in this file does not belong to this PR. Can we remove un-needed code?